package.json

```diff
@@ -50,8 +50,12 @@
     "test": "istanbul cover -x '**/*.test.js' node_modules/mocha/bin/_mocha '!(node_modules)/**/*.test.js' -- -R spec --recursive",
     "lint": "eslint .",
     "docs": "node scripts/generate-readme.js",
-    "simple-integration-test": "mocha tests/integration/simple-integration-test",
-    "complex-integration-test": "mocha tests/integration/all"
+    "simple-integration-test": "jest --maxWorkers=5 simple-suite",
```

Comment:
Just wondering if we should nest both tests inside of a `integration` directory and call this one `simple` and the other one `complex` since both are integration tests...

```
tests/integartion/simple/service-lifecycle.js
tests/integration/complex/...
```